epsilon-etl/epsilon-etl_activity.json

```diff
@@ -10,7 +10,7 @@
 
             "icon": "etl",
 
-            "tools": [ "http://127.0.0.1:8070/epsilon_tool.json",
+            "tools": [ "http://127.0.0.1:8070/tools",
```

Comment:
Is this not a `.json` file?

Reply:
The json file is mapped to this url without an extension so is correct.